style.scss

```diff
@@ -222,19 +210,286 @@ body>footer {
   @include stripe {
     background-color: #d0d0e8;
   }
-}
-
+}
+
+// =======================================================================================
+// community.html
+
+
+body>#community {
+  >header {
+  min-height: 150px;
+  color: white;
+
+  @include stripe {
+    background-color: #1b4460;
+  }
+
+    >h1 {

```

Comment:
This is nitpicky and not super-important, but you have some inconsistent indentation, and this can make it hard to read the code. Indentation is a lot more visible than the actual `{}` braces, so over time you'll learn to rely on it when jumping around quickly, but if it's inconsistent then you won't be able to rely on it.